add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
**Observations**: The code directly uses `$_POST` and `$_GET` variables without any validation or sanitization, which makes it vulnerable to SQL injection attacks.

**Recommendations**: Use prepared statements with parameterized queries to prevent SQL injection. Additionally, validate and sanitize input data before processing.

**Example**:
```php
if ($_SERVER['REQUEST_METHOD'] == 'POST') {
    $title = mysqli_real_escape_string($connection, $_POST['title']);
    $author = mysqli_real_escape_string($connection, $_POST['author']);
    $year = (int)$_POST['year'];

    $query = "INSERT INTO books (title, author, year) VALUES (?, ?, ?)";
    $stmt = mysqli_prepare($connection, $query);
    mysqli_stmt_bind_param($stmt, 'ssi', $title, $author, $year);
    if (mysqli_stmt_execute($stmt)) {
        echo "Book added successfully!";
    } else {
        echo "Error: " . mysqli_stmt_error($stmt);
    }
}
```